backend/README.md

```diff
@@ -27,14 +27,20 @@ the regular base application.
 The project requires a couple of environment variables to run, if you want to develop on this codebase.
 Setting values for these variables can be done with a method to your own liking.
 
-| Variable          | Description                                                    |
-|-------------------|----------------------------------------------------------------|
-| DB_HOST           | Url of where the database is located                           |
-| POSTGRES_USER     | Name of the user, needed to login to the postgres database     |
-| POSTGRES_PASSWORD | Password of the user, needed to login to the postgres database |
-| POSTGRES_HOST     | IP adress of the postgres database                             |
-| POSTGRES_DB       | Name of the postgres database                                  |
-| API_HOST          | Location of the API root                                       |
+| Variable                               | Description                                                                         |
+|----------------------------------------|-------------------------------------------------------------------------------------|
+| DB_HOST                                | Url of where the database is located                                                |
+| POSTGRES_USER                          | Name of the user, needed to login to the postgres database                          |
+| POSTGRES_PASSWORD                      | Password of the user, needed to login to the postgres database                      |
+| POSTGRES_HOST                          | IP adress of the postgres database                                                  |
```

Comment:
Not necessarily an IP address. What's the difference between this and `DB_HOST`

Reply:
DB_HOST isn't actually needed so can be removed

Reply:
https://github.com/SELab-2/UGent-3/pull/357/commits/115417609c6a0b358e1ae30ee816d1097ee1334d